mapper/mapper_test.go

```diff
@@ -642,7 +643,7 @@ func TestCreatePreviousReleasesPage(t *testing.T) {
 		So(err, ShouldBeNil)
 
 		Convey("When CreatePreviousReleasesPage is called", func() {
-			sp := CreatePreviousReleasesPage(cfg, req, mdl, validatedQueryParams, respC, englishLang, respH, "", &topicModels.Navigation{}, "", cache.Topic{}, nil, respZ)
+			sp := CreatePreviousReleasesPage(cfg, req, mdl, validatedQueryParams, respC, englishLang, respH, "", &topicModels.Navigation{}, "", cache.Topic{}, nil, respZ, []zebedee.Breadcrumb{})
```

Comment:
Can we test the breadcrumb mapping here? And latest release?

Reply:
👍